data/variabledefs-rescue.csv

```diff
@@ -0,0 +1,136 @@
+gas,sector,unit,include_in_total,griddinglevel,proxy_path,proxy_name,proxy_sector
+BC,Agricultural Waste Burning,Mt BC/yr,TRUE,country,openburning_BC.nc,BC_em_openburning,AWB
+BC,Agriculture,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,AGR
+BC,Aircraft,Mt BC/yr,TRUE,global,aircraft_BC.nc,BC_em_AIR_anthro,AIR
+BC,Energy Sector|Modelled,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,ENE
+BC,Energy Sector|Non-Modelled,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,ENE
+BC,Forest Burning,Mt BC/yr,TRUE,country,openburning_BC.nc,BC_em_openburning,FRTB
+BC,Grassland Burning,Mt BC/yr,TRUE,country,openburning_BC.nc,BC_em_openburning,GRSB
+BC,Industrial Sector,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,IND
+BC,International Shipping,Mt BC/yr,TRUE,global,shipping_BC.nc,BC_em_anthro,SHP
+BC,Peat Burning,Mt BC/yr,TRUE,country,openburning_BC.nc,BC_em_openburning,PEAT
+BC,Residential Commercial Other,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,RCO
+BC,Solvents Production and Application,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,SLV
+BC,Transportation Sector,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,TRA
+BC,Waste,Mt BC/yr,TRUE,country,anthro_BC.nc,BC_em_anthro,WST
+CH4,Agricultural Waste Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,AWB
+CH4,Agriculture,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,AGR
+CH4,Aircraft,Mt CH4/yr,TRUE,global,aircraft_CH4.nc,CH4_em_AIR_anthro,AIR
+CH4,Energy Sector|Modelled,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,ENE
+CH4,Energy Sector|Non-Modelled,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,ENE
+CH4,Forest Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,FRTB
+CH4,Grassland Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,GRSB
+CH4,Industrial Sector,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,IND
+CH4,International Shipping,Mt CH4/yr,TRUE,global,shipping_CH4.nc,CH4_em_anthro,SHP
+CH4,Peat Burning,Mt CH4/yr,TRUE,country,openburning_CH4.nc,CH4_em_openburning,PEAT
+CH4,Residential Commercial Other,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,RCO
+CH4,Solvents Production and Application,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,SLV
+CH4,Transportation Sector,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,TRA
+CH4,Waste,Mt CH4/yr,TRUE,country,anthro_CH4.nc,CH4_em_anthro,WST
+CO2,Aggregate - Agriculture and LUC,Mt CO2/yr,TRUE,global,,,

```

Comment:
On mattermost, we agreed to rename this variable `Deforestation and other LUC`. It should be treated in the same way as the aforestation variable (same harm/ds/grid approach and pattern)